build/pack.js

```diff
@@ -63,6 +64,7 @@ module.exports = function (opts) {
     utils.collect(_c.LIB, lib);
     utils.collect(_c.DEVICES, devicesCSS, matches(".css"));
     utils.collect(_c.UI, overlays, matches("overlay.html"));
+    utils.collect(_c.UI, overlayStyles, matches("overlay.css"));
```

Comment:
Oh – I see, this is something new we added for the XDK.

In our Ripple derivative, we currently have 20 overlay.css files in subfolders of the lib/client/ui/plugins folder.  This allows us to avoid putting style either inline in the panel.html file or in the global ripple.css file, but still have some richer appearance.

I guess this part of the PR is useless but harmless as it is; I expect I expect I will want to add this capability to ripple later on.  Of course, when we got to the Ripple 2.0 framework, the simulation controls will be represented differently.

    Julian

From: Tim Barham [mailto:notifications@github.com]
Sent: Wednesday, April 15, 2015 7:58 PM
To: apache/incubator-ripple
Cc: Horn, Julian C
Subject: Re: [incubator-ripple] Made jake [lint, build, test] work on windows (#47)


In build/pack.js<https://github.com/apache/incubator-ripple/pull/47#discussion_r28474578>:

> @@ -63,6 +64,7 @@ module.exports = function (opts) {

>      utils.collect(_c.LIB, lib);

>      utils.collect(_c.DEVICES, devicesCSS, matches(".css"));

>      utils.collect(_c.UI, overlays, matches("overlay.html"));

> +    utils.collect(_c.UI, overlayStyles, matches("overlay.css"));

Can you explain why we need this? There aren't currently any overlay.css files.

—
Reply to this email directly or view it on GitHub<https://github.com/apache/incubator-ripple/pull/47/files#r28474578>.